backend/Application/Mapper/MapperProfile.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Application.DTOs.Admin;
+using AutoMapper;
+using Domain.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Application.Mapper
+{
+    public class MapperProfile : Profile
+    {
+        public MapperProfile()
+        {
+            #region Admin
+            CreateMap<Admin, UpdateAdminDTos>()
```

Comment:
Remove all update related methods, validations and mapping, let's worry about that later, so we don't create anything that we will not use.